cli/src/pcluster/config/config_patch.py

```diff
@@ -189,18 +189,19 @@ def _compare_list(self, base_section, target_section, param_path, data_key, fiel
                     )
                 )
         # Then, compare all non visited base sections vs target config.
-        for base_nested_section in base_section.get(data_key, []):
-            if not base_nested_section.get("visited", False):
-                self.changes.append(
-                    Change(
-                        param_path,
-                        data_key,
-                        base_nested_section,
-                        None,
-                        change_update_policy,
-                        is_list=True,
+        if base_section:
```

Comment:
because when the _test_storage from test_config_patch.py runs, it calls _compare_list on with a base_section of None, because it tries to go into data_repository_association since it is a nested attribute, but data_repository_association is None, which makes the test fail without this change.

Reply:
Thank you.